pkg/metadata/image_metadata_test.go

```diff
@@ -27,7 +27,7 @@ import (
 func TestImageMetadataStore(t *testing.T) {
 	imageMetadataMap := map[string]*ImageMetadata{
 		"1": {
-			ID:   "1", // TODO(mikebrow): fill
+			ID:   "1",
```

Comment:
The TODO means that we should fill in all the fields of the image metadata in the test.
I'll do that.